user-profile-dialog.html

```diff
@@ -224,21 +230,25 @@
         _closeUserProfileDialog: function(event) {
           if (event.detail.confirmed) {
             this.saveData();
+          } else {
+            this._resetWorkingProfile();
           }
         },
         _mapIds: function(profile) {
-          var availableCountryIds = _.map(this.profile.countries_available, 'id');
-          var availableGroups = _.map(this.profile.groups, 'id');
+          var availableCountryIds = _.map(this.workingProfile.countries_available, 'id');
+          var availableGroups = _.map(this.workingProfile.groups, 'id');
 
           this.set('availableCountryIds', availableCountryIds);
           this.set('availableGroups', availableGroups);
         },
+        // Is this needed?
         resetData: function() {
           this.set('profile', JSON.parse(JSON.stringify(this.userProfileModel)));
         },
         saveData: function() {
-          this.fire('hacky-save', {profile: this.profile});
+          this.fire('hacky-save', {profile: this.workingProfile});
         },
+        // where is this used?
         _convertOffices: function(data) {
           var offices = data.map(function(office) {
```

Comment:
@marko911 Are methods `_convertOffices`, `_centerDialog` and `resetData` being used? I can't find any calls to them